cowprotocol/lookup_tables/daily_stable_tokens_5469456.sql

```diff
@@ -0,0 +1,41 @@
+-- this query returns a daily list of token addresses which fit into a few price stability criteria over the previous 30d
```

Comment:
could you maybe describe these stability criteria (if this is easy) up here so that there is no need to look at the code?